pylmod/tests/test_gradebook.py

```diff
@@ -17,6 +18,7 @@
 from pylmod.tests.common import BaseTest
 
 
+@ddt
```

Comment:
It parameterizes the tests to run with multiple sets of data. You need the `@ddt` on the class and `@data` on whichever test you parmeterize